jans-client-api/gen-client/src/test/resources/testng.xml

```diff
@@ -21,28 +21,28 @@
                value="{'resources':[{'path':'/GetAll','conditions':[{'httpMethods':['GET'],'scopes':['http://photoz.example.com/dev/actions/a1','http://photoz.example.com/dev/actions/a2','http://photoz.example.com/dev/actions/a3'],'ticketScopes':['http://photoz.example.com/dev/actions/a1','http://photoz.example.com/dev/actions/a2','http://photoz.example.com/dev/actions/a3'],'scope_expression':{'rule':{'and':[{'or':[{'var':0},{'var':1}]},{'var':2}]},'data':['http://photoz.example.com/dev/actions/a1','http://photoz.example.com/dev/actions/a2','http://photoz.example.com/dev/actions/a3']}}]}]}"/>
 
     <!-- main class to set up and tear down suite -->
-   <test name="SetUp" enabled="true">
-        <classes>
-            <class name="io.swagger.client.api.SetUpTest"/>
-        </classes>
-    </test>
+<!--   <test name="SetUp" enabled="true">-->
+<!--        <classes>-->
+<!--            <class name="io.swagger.client.api.SetUpTest"/>-->
+<!--        </classes>-->
+<!--    </test>-->
```

Comment:
Adding ticket to enable these tests back based on swagger generated client. 

https://github.com/JanssenProject/jans/issues/1531